libs/cart/testing/src/factories/magento/money.factory.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { TestBed } from '@angular/core/testing';
+
+import { MagentoMoneyFactory } from './money.factory';
+import { MagentoMoney } from '@daffodil/cart';
+
+describe('Cart | Testing | Factories | MoneyFactory', () => {
+  let cartItemFactory: MagentoMoneyFactory;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [MagentoMoneyFactory]
+    });
+
+    cartItemFactory = TestBed.get(MagentoMoneyFactory);
+  });
+
+  it('should be created', () => {
+    expect(cartItemFactory).toBeTruthy();
+  });
+
+  describe('create', () => {
+    let result: MagentoMoney;
+
+    beforeEach(() => {
+      result = cartItemFactory.create();
+    });
+
+    it('should return a Money with all required fields defined', () => {
+      expect(result.currency).toBeDefined();
+      expect(result.value).toBeDefined();
+    });
+  });
+
+  describe('createMany', () => {
```

Comment:
I think we can stop testing the createMany function in all our factories, because that's tested in the DaffModelFactory. This test can be removed

Reply:
Changed in [`79c33e7`](https://github.com/graycoreio/daffodil/pull/657/commits/79c33e78f740b04bcabb1b51272dba8caff3e812).